__tests__/analysis.test.js

```diff
@@ -0,0 +1,27 @@
+import analyze from '../src/analysis';
+
+test('should return an object', () => {
+  expect(typeof analyze([1, 8, 3, 4, 2, 6])).toBe('object');
+});
+
+test('takes in an array and returns an object containing avg, min, max & length', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6])).toEqual({
+    average: 4, min: 1, max: 8, length: 6,
+  });
+});
+
+test('should have property average', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).average).toBe(4);
```

Comment:
- [ ] [OPTIONAL] Test what happens in an empty array here and make sure the error/result is handled properly.